pkg/common/log/common.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package log
+
+import "go.uber.org/zap"
+
+type loggerFunc func(msg string, fields ...zap.Field)
+
+// CloseResponseBodyError outputs a 'close response body' error log to the given logger.
+// Note: This function is an example and is not currently in use
```

Comment:
Do you plan on using this in the future? If so, you should omit the `and is not currently in use` part so you don't have to remember to come back here and update it